test/files/run/splain.check

```diff
@@ -0,0 +1,62 @@
+newSource1.scala:13: error: implicit error;
+!I e: II
+ImplicitChain.g invalid because
+!I impPar3: I1
+⋮
+――ImplicitChain.i1 invalid because
+  !I impPar7: I3
+  implicitly[II]
+            ^
+newSource1.scala:6: error: type mismatch;
+  L|R
+  f(new L)
+    ^
+newSource1.scala:7: error: implicit error;
+!I e: F[Arg]
+
+  implicitly[F[Arg]]
+            ^
+newSource1.scala:4: error: implicit error;
+!I ec: ExecutionContext
+  Cannot find an implicit ExecutionContext. You might add
```

Comment:
In the base case when there were no discarded alternatives, the `!I ...` line doesn't seem to add any value?

Also, the default message skips the `could not find implicit value` part (`implicit error` here) when the target has an `@implicitNotFound`.

Reply:
marking as resolved since improvement here would be welcome but it's not a blocker